man/systemctl.xml

```diff
@@ -1772,17 +1772,20 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
       </varlistentry>
 
       <varlistentry>
-        <term><option>-i</option></term>
-        <term><option>--ignore-inhibitors</option></term>
+        <term><option>--respect-inhibitors=</option></term>
 
         <listitem>
-          <para>When system shutdown or a sleep state is requested, ignore inhibitor locks. Applications can establish
-          inhibitor locks to avoid that certain important operations (such as CD burning or suchlike) are interrupted
-          by system shutdown or a sleep state. Any user may take these locks and privileged users may override these
-          locks. If any locks are taken, shutdown and sleep state requests will normally fail (unless privileged) and a
-          list of active locks is printed. However, if <option>--ignore-inhibitors</option> is specified, the
-          established locks are ignored and not shown, and the operation attempted anyway, possibly requiring
-          additional privileges.</para>
+          <para>When system shutdown or sleep state is request, this option controls how to deal with inhibitor locks. It takes one of <literal>auto</literal>, <literal>yes</literal> or <literal>no</literal>.
+          Defaults to <literal>auto</literal>, which will behave like <literal>yes</literal> for interactive requests
+          and <literal>no</literal> for non-interactive requests.
+          <literal>yes</literal> will let the request respecting inhibitor locks.
+          <literal>no</literal> will let the request ignoring inhibitor locks.
+          </para>
+          <para>Applications can establish inhibitor locks to avoid that certain important operations (such as CD burning or suchlike) are interrupted by system shutdown or a sleep state. Any user may take these locks and privileged users may override these locks.
+          If any locks are taken, shutdown and sleep state requests will normally fail (unless privileged) and a
+          list of active locks is printed.
+          However, if <literal>no</literal> is specified or <literal>auto</literal> is specified on a non-interactive requests, the established locks are ignored and not shown, and the operation attempted anyway, possibly requiring additional privileges.
+          May be overriden by <option>--force</option>.</para>
```

Comment:
please rebreak this to be 109 chars wide. (this is what we use for xml docs like this, see .dir-locals.el and .vimrc — if you use emacs or vim this should be the default fill column to make things easy)